File: program/convert-feet-to-inches/convertFeetToInches.scala
import scala.io.StdIn._;

object feetToInches {
  def main(args:Array[String]):Unit={
    println("Enter the feet value : ");
    var feet = readLine();
    var inches = feet.toInt * 12;
    println(feet+" feet is "+inches+" inches");
  }
}

File: program/program/convert-feet-to-inches/convert_feet_to_inches.pl
print "Enter the feet value : ";
my $inch = <STDIN> ;

my $res = feet2Inches($inch);
print "Inch conversion for the given feet : $res inches";

sub feet2Inches
{
    my ($x) = @_;
    my $res = $x * 12;
    return $res ;   
}
